tools/ete/test-data/lineage-compress-lower.txt

```diff
@@ -29,7 +29,7 @@ Nomascus leucogenys	Eukaryota	Chordata	Mammalia	Euarchontoglires	Hominoidea	Noma
 Pongo abelii	Eukaryota	Chordata	Mammalia	Euarchontoglires	Hominoidea	Pongo	Pongo abelii
 Gorilla gorilla gorilla	Eukaryota	Chordata	Mammalia	Euarchontoglires	Hominoidea	Gorilla	Gorilla gorilla
 Pan troglodytes	Eukaryota	Chordata	Mammalia	Euarchontoglires	Hominoidea	Pan	Pan troglodytes
-Homo sapiens	Eukaryota	Chordata	Mammalia	Euarchontoglires	Hominoidea	Homo	Homo sapiens
+9606	Eukaryota	Chordata	Mammalia	Euarchontoglires	Hominoidea	Homo	Homo sapiens
```

Comment:
I'm not sure it's good that this and the following output files are changed, probably need to modify `ete_lineage_generator.py` to print the species name instead of the input taxid in the first column?

Reply:
I also noted this. The change is because the first column is just a copy of the input. I liked this because it gives a mapping between the original inputs and the ete outputs.

Should be easy to change it if you like. But we can also document it better. 

In most use cases people will input either taxids of species names.

Reply:
@anilthanki Any opinion on this?

Reply:
Hi @nsoranzo can we interpret no answer as positive feedback?

Reply:
Hi @anilthanki: any chance the get feedback? If not, can we just have a decision on how to continue here?

Reply:
Hi @bernt-matthias I did have a look before and try to remember what It supposed to do but TBH its been long I worked on it so I couldnt really make a comment..

I will try to review it again this weekend and make a comment by monday

Apologies for this